src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -371,6 +373,17 @@ private ImmutableArray<Diagnostic> GetFunctionDiagnostics(Diagnostic diagnostic)
             return ImmutableArray<Diagnostic>.Empty;
         }
 
+        private static bool IsCorrectType(string systemType, string queryType)
```

Comment:
This doesn't have any callers, so it should be removed (which would eliminate any changes to this file)